package.json

```diff
@@ -13,17 +13,17 @@
   },
   "scripts": {
     "build": "tsc",
-    "test": "npm run build && mocha && npm run lint",
+    "test": "yarn run build && mocha && yarn run lint",
```

Comment:
you can do `yarn build && mocha && yarn lint`. `npm` requires the use of `run` keyword before the script name.